clarity/enhancer/nalr.py

```diff
@@ -154,7 +154,7 @@ def build(
             delay
         """
         if cfs is None:
-            cfs = np.array([250, 500, 1000, 2000, 3000, 6000])
+            cfs = np.array([250, 500, 1000, 2000, 4000, 6000])
```

Comment:
These default cfs are the audiometric cfs stored in the constructor, i.e. there is no need to interpolate the hl value, so the correct implementation would be

```python
if cfs is not None:
    hl =  self.hl_interp(np.array(hl), np.array(cfs))
```